public/main.js

```diff
@@ -1,12 +1,16 @@
 $(document).on('ready', function(){
 	submitAnswer();
+    displayMessage();
```

Comment:
@podviaznikov 

i have renamed it. but if it will be bad, help me maybe, because i dont know.